### ISSUE TITLE
Chess Game

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/Play-With-Python#8]
### Play With Python Repository (Proposing new issue)
:red_circle: **Project Title** : Chess Game</br>
:red_circle: **Aim** : Create a Chess Game using Python programming language.</br>

**********************************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by kunalgupta10** [comment]
Full name : Gupta Kunal
GitHub Profile Link : github.com/kunalgupta10
Email ID :  kunalguptakrg10@gmail.com
Participant ID (if applicable):
Approach for this Project : I have recently done projects on ML and DL and I wanted to make a core python project 
I am going to use PYGAME module of python for building the chess game

**Comment by abhisheks008** [comment]
Thanks @kunalgupta10 for showing your interest. Unfortunately, this repository will kick start its journey in an open source event, after the launch of this open source repo, you can start contributing here. I'll notify once the launch is done.

Have a great day.

**Comment by Siddharth9146** [comment]
Full name : Siddharth Rajopadhye
GitHub Profile Link : https://github.com/Siddharth9146
Email ID : rajopadhyesiddharth@gmail.com
Participant ID (if applicable):
Approach for this Project : I am currently learning AI and I want to make a chess game with a strong AI opponent in Python(pygame)
What is your participant role? (Mention the Open Source program)

**Comment by abhisheks008** [comment]
Issue assigned to you @Siddharth9146 
Go ahead!

**Comment by abhisheks008** [comment]
Updates please @Siddharth9146
